docs/source/running-autometa.rst

```diff
@@ -2,108 +2,271 @@
+
+.. todo::
+    Need to alter the url for ``parameters.config`` after the merge is done.
+
+Data Inputs
+^^^^^^^^^^^
+
+.. note::
+    Data inputs must be wrapped in 'single quotes' or "double quotes"
+
+The first and the most important thing that you need to alter is the paths to various input and output files.
+
+*params.metagenome* : Change this to point to your metagenome assembly. Eg. ``"/media/bigdrive1/sidd/test_data/78mbp_metagenome.fna"``
+
+*params.interim* : Change this to point to where you want the interim results to be stored. Eg. ``"/media/bigdrive1/sidd//test_data/interim/"``. Nextflow_ will create a new directory if absent.
+
+*params.processed* : Change this to point to where you want the final results to be stored. Eg. ``"/media/bigdrive1/sidd/test_data/processed/"``. Nextflow_ will create a new directory if absent.
+
+.. code-block:: bash
+
+    // Find this section of code in parameters.config
+    params.metagenome = "<path/to/assembly.fna>" 
+    params.interim = "<path/to/interim_results/>" 
+    params.processed = "<path/to/processed_results/>"
+
+You can also input multiple asseblies at once with the help of wildcards. In the below example all the files with extension "fasta" would be taken as input by nextflow_.
+
+.. code-block:: bash
+
+    // Find this section of code in parameters.config
+    params.metagenome = "<path/to/assembly_dir/*.fasta>" 
+    params.interim = "<path/to/interim_results/>" 
+    params.processed = "<path/to/processed_results/>"
+
+.. note::
+    Wildcard characters will only be interpreted when "double quotes" are used
+
+Database directory
+^^^^^^^^^^^^^^^^^^
+
+Autometa uses the following NCBI databses throughout its pipeline:
+
+- Non-redundant `nr database <ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz>`_
+- `prot.accession2taxid.gz <ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz>`_
+- *nodes.dmp*, *names.dmp* and *merged.dmp* from `taxdump tarball <ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz>`_ 
+
+If you are running autometa for the first time you'll have to download these databases. Use ``autometa-update-databases --update-ncbi``. This will download the databases to the default path. You can check the default paths using ``autometa-config --print``. If you need to change the default download directory you can use ``autometa-config --section databases --option ncbi --value <path/to/new/ncbi_database_directory>``. See ``autometa-update-databases -h`` and ``autometa-config-h`` for full list of options.
```

Comment:
This is kind of making me think we should just have a default path be the current directory if the user has not yet configured their databases section.

I.e. Without configuration

```bash
autometa-update-databases --update-ncbi
```

creates databases and specified sub-directories

1. mkdir -p ./databases/ncbi
2. downloads and formats ncbi files to `./databases/ncbi`

Maybe we need to rethink? Also would `autometa-databases` be a better entrypoint name?